### PR TITLE
Updates RichTextBlock

### PR DESCRIPTION
**Pull request by gustaveen**
## Describe your changes
Update RichTextBlock according to new design

## Justify why they are needed
Needed to migrate content from market-web
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by gustaveen** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2220** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2220" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2220?utm_source=stack-comment).

File: apps/store/src/blocks/RichTextBlock/RichTextBlock.styles.tsx [hunk 1]
```diff
@@ -156,41 +148,31 @@ export const richTextStyles = css(
...
-      },
-
-      h3: {
-        marginTop: theme.space.xxxl,
-      },
-
-      h4: {
-        marginTop: theme.space.xxxl,
+    [mq.md]: {
+      '&[data-small-text=false]': {
```
Comment (alebedev) [inline apps/store/src/blocks/RichTextBlock/RichTextBlock.styles.tsx:157]: Can we use some other way?  We may remove `data-small-text` in the future and get unexpected style changes

**Review by alebedev — APPROVED** [review]

File: apps/store/src/blocks/RichTextBlock/RichTextBlock.styles.tsx (hunk 1 above)
Comment (gustaveen) [reply apps/store/src/blocks/RichTextBlock/RichTextBlock.styles.tsx:152]: Would it be better to do it the other way around? Having `data-large-text=true` @alebedev

Comment (alebedev) [reply apps/store/src/blocks/RichTextBlock/RichTextBlock.styles.tsx:152]: Yes, since small text is probably a default

**Comment by gustaveen** [comment]
@gustaveen merged this pull request with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/2220).

**Pull request merged**